### ISSUE TITLE
Error updating to latest Alpha

### ISSUE THREAD

**Issue by ehollig** [nvaccess/nvda#10270]
# Steps to reproduce:
1. Create a portable copy of alpha-18330,b4db7dca
2. Go to NVDA Help > Check for update
3. Try to update to alpha-18737,e9cfa1b1

### Actual behavior:
NVDA is unable to remove a temporary file and fails the portable update.

### Expected behavior:
NVDA updates to latest Alpha.

#### NVDA installed/portable/running from source:
Portable

#### NVDA version:
Version: alpha-18330,b4db7dca

#### Windows version:
Windows 10,1809

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### Additional info ####
WARNING - updateCheck.initialize (11:09:53.312) - MainThread (9480):
Unable to remove old update file [User path]\NVDA Alpha\userConfig\updates\nvda_update_tqify3hk.exe
Traceback (most recent call last):
  File "updateCheck.pyc", line 782, in initialize
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: [User path]\NVDA Alpha\\userConfig\\updates\\nvda_update_tqify3hk.exe'

Traceback (most recent call last):
  File "gui\installerGui.pyc", line 359, in doCreatePortable
  File "gui\__init__.pyc", line 900, in __init__
  File "threading.pyc", line 1077, in name
AssertionError: Thread.__init__() not called

**Comment by ehollig** [comment]
Duplicate of #10266
